lib/std/fs/file.zig

```diff
@@ -226,6 +226,13 @@ pub const File = struct {
     /// Test whether ANSI escape codes will be treated as such.
     pub fn supportsAnsiEscapeCodes(self: File) bool {
         if (builtin.os.tag == .windows) {
+            if (!os.isatty(self.handle)) return false;
```

Comment:
This is redundant and should be removed, the implementation of `os.isatty` on Window is:

```zig
        if (isCygwinPty(handle))
            return true;

        var out: windows.DWORD = undefined;
        return windows.kernel32.GetConsoleMode(handle, &out) != 0;
```